src/main/java/com/forest/mytopmovies/config/CacheConfig.java

```diff
@@ -0,0 +1,27 @@
+package com.forest.mytopmovies.config;
+
+import com.github.benmanes.caffeine.cache.Caffeine;
+import org.springframework.cache.CacheManager;
+import org.springframework.cache.annotation.EnableCaching;
+import org.springframework.cache.caffeine.CaffeineCacheManager;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.concurrent.TimeUnit;
+
+@Configuration
+@EnableCaching
+public class CacheConfig {
+    @Bean
+    public Caffeine caffeineConfig() {
```

Comment:
Maybe think about the generics that should fill in Caffeine, i.e: Caffeine<Object, Object>